Cargo.toml

```
@@ -19,3 +19,6 @@ validator = "0.16.0"
 validator_derive = "0.16.0"
 tokio-postgres = "0.7.8"
 sea-orm = { version = "0.11.3", features = [ "sqlx-postgres", "runtime-actix-native-tls", "macros" ] }
+rust-argon2 = "0.8"
+rand_core = { version = "0.6", features = ["std"] }
+rand = "0.8.4"
```

использование старых версий библиотек
крэйт rand_core от версии 0.6 и ниже версии 0.6.2 содержит критические уязвимости
```suggestion
rust-argon2 = "1.0.0"
rand_core = { version = "0.6.4", features = ["std"] }
rand = "0.8.5"
```